zipline/lib/_windowtemplate.pxi

```diff
@@ -138,6 +145,8 @@ cdef class AdjustedArrayWindow:
         new_out = asanyarray(self.data[anchor - self.window_length:anchor])
         if view_kwargs:
             new_out = new_out.view(**view_kwargs)
+        if self.rounding_places is not None and new_out.dtype is dtype('float64'):
```

Comment:
Good call on the `np.issubdtype`, will change.

> Also any reason not to just import float64 directly?

I've seen throughout zipline we tend to favor using `dtype('float64')` as seen [here](https://github.com/quantopian/zipline/blob/master/zipline/utils/numpy_utils.py#L35), so I was just following that convention.